src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(number, result) {
+      return result + number;
+    },
+
+    subtract(number, result) {
+      return result - number;
+    },
+
+    divide(number, result) {
+      if (result === 0) {
+        throw new Error('value is not correct');
```

Comment:
Try to explain errors more
Also, you divide by `number`, but check `result` for being 0.

```suggestion
        throw new Error('value is not correct - division by 0');
```